src/app/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import TextInput from '../../components/TextInput/TextInput';
+import SubmitButton from '../../components/SubmitButton/SubmitButton';
+import Typography from '../../components/Typography/Typography';
+import { Link } from 'react-router-dom';
+import styles from './LoginPage.module.css';
+
+export default function LoginPage(): JSX.Element {
+  return (
+    <main className={styles.login}>
+      <h2>
+        <Typography size="l">Einloggen</Typography>
+      </h2>
+      <TextInput id="name" inputType="text">
+        Name
+      </TextInput>
```

Comment:
Sorry, I should've spotted this earlier, but I think it would look better if `TextInput` had a `label` prop to assign label text. This is not wrong, but it looks misleading, since input elements aren't allowed to have children. Probably an issue for later though :)

Reply:
Will be resolved when I finally stop fooling around with CSS and start with actual functions. Thanks for the head's up!